main/handlers/scrape.go

```
@@ -74,109 +105,433 @@ func HandleScrapeDocsRaw(logger *log.Logger, pgxConn *pgxpool.Pool, supabaseURL
+				logger.Printf("Failed to get markdown: %v", err)
+				continue
+			}
+
+			title, err := helpers.GetTitleFromJinaMarkdown(logger, markdown)
+			if err != nil {
+				logger.Printf("Failed to get title: %v\n%s", err, markdown)
+				continue
+			}
+
+			var pageID int
+			err = pgxConn.QueryRow(r.Context(), "INSERT INTO pages (url_id, title) VALUES ($1, $2) RETURNING id", urlID, title).Scan(&pageID)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)